delta/src/main/scala/ch/epfl/bluebrain/nexus/delta/Main.scala

```diff
@@ -169,7 +166,8 @@ object Main {
         ArchiveCache[Task].runSyncUnsafe()(Scheduler.global, pm)
       )
     implicit val pc: ProjectCache[Task]           = cache.project
-    implicit val projectResolution                = ProjectResolution.task(repo, cache.resolver, cache.project, acls, saCaller)
+    implicit val projectResolution                =
```

Comment:
Formatting is weird here

Reply:
Yeah. It is scalafmt formatting. Not a big deal